Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -94,6 +97,12 @@
     <ProjectReference Include="..\Flow.Launcher.Plugin\Flow.Launcher.Plugin.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Images\app.ico">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
```

Comment:
this is not needed right since you are including all .ico extension

Reply:
oh yes